.github/workflows/ci.yaml

```diff
@@ -55,7 +55,27 @@ jobs:
         run: make build
       - name: docker build
         run: make docker
-
+  licenses-lint:
+    name: Licenses Lint
+    timeout-minutes: 40
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.20.x
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: generate license mirror
+        run: |
+          make mirror-licenses
```

Comment:
It is not enough, we should validate the licenses dir is not changed. Please refer `gen-check.sh`